Tasks/AzureRmWebAppDeployment/azurermwebappdeployment.ts

```diff
@@ -123,6 +123,7 @@ async function main() {
     }
     catch(error) {
         isDeploymentSuccess = false;
+        console.log("##vso[task.logissue type=error;code=AASD_Task_InternalError;]" + error);	
```

Comment:
Can you please create error type to be Enum in common folder so we can reuse it .